packages/agent-graph/src/hooks/useGraphCamera.ts

```
@@ -118,47 +132,55 @@ export function useGraphCamera(): UseGraphCameraResult {
+        minY = Infinity,
+        maxX = -Infinity,
+        maxY = -Infinity;
+      for (const n of nodes) {
+        const x = n.x ?? 0;
+        const y = n.y ?? 0;
+        const pad =
+          n.kind === 'task'
+            ? TASK_PILL.width / 2
+            : n.kind === 'lead'
+              ? NODE.radiusLead
+              : NODE.radiusMember;
```

ESLint

[error] Extract this nested ternary operation into an independent statement.

(sonarjs/no-nested-conditional)